.github/workflows/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Siti che contengono link rotti
+labels: housekeeping
```

Comment:
We don't have this label (and [bug](https://github.com/italia/awesome-italian-public-datasets/labels/bug) sounds too generic), but we can create one. I'd use something specific like `broken-links`.